formspree/app.py

```diff
@@ -2,18 +2,31 @@
 from flask import g
 from flask.ext.sqlalchemy import SQLAlchemy
 from flask.ext.login import LoginManager, current_user
-import fakeredis, redis
+import redis
 import settings
 
+
 DB = SQLAlchemy()
-if settings.TESTING:
-    REDIS = fakeredis.FakeStrictRedis()
-else:
-    REDIS = redis.StrictRedis.from_url(settings.REDIS_URL)
+_redis = None
+
+
+def REDIS():
+    ''' function to acquire the global redis connection '''
+    global _redis
+    if not _redis:
+        _redis = get_redis()
+    return _redis
```

Comment:
The "pythonic" argument is persuasive. Though for the sake of argument, aside from aesthetics I fail to see the benefit of importing a module attribute whose value is instantiated at load time, vs calling a singleton constructor that lazily instantiates the object. Both provide access to a single app-wide instance. 

However, comparing my code to PHP... that's just mean.